Makefile

```diff
@@ -15,3 +15,8 @@ docs:
 
 postgres:
 	python setup.py build_postgres
+
+
+pygments:
+	edb gen-meta-grammars edgeql > edb/edgeql/pygments/meta.py
+	edb gen-meta-grammars > edb/eschema/pygments/meta.py
```

Comment:
is eschema default? why?  There should be no default language for 'gen-meta-grammars'

Reply:
ah, I see, there's a default "all languages" -- no need for that default. One should always specify what they need (otherwise what happens if we add 3 new languages and the tool starts to generate shit that nobody expected it to generate)

Reply:
Default is *everything*. Currently it's edgeql + eschema, but in the future this argument will be irrelevant since we'll just have one single grammar, most likely.

Reply:
just drop the default; always require at least one language.

Reply:
OK, fair point about explicit specification of the languages..